samples/apps/copilot-chat-app/webapi/CopilotChat/Skills/SemanticSkills/Chat/config.json

```diff
@@ -0,0 +1,81 @@
+{
+  "schema": 1,
+  "description": "Generate a chat response from the underlying LLM",
+  "type": "completion",
+  "completion": {
```

Comment:
nit: probably worth documenting why these values are selected

Reply:
Besides max token, these values are default values. We also have documented it [here](https://learn.microsoft.com/en-us/semantic-kernel/ai-orchestration/semantic-functions?tabs=Csharp).

Reply:
Then can we have a comment that links to the link and optionally another one that explains why the max token is selected (since now we have multiple config files with different max tokens values?

Reply:
I'll add a tiny readme to the Semantic Skills folder